website/content/commands/services/deregister.mdx

```diff
@@ -38,23 +38,23 @@ in HCL or JSON format.
 This flexibility makes it easy to pair the command with the
 `services register` command since the argument syntax is the same.
 
-#### API Options
+#### Command Options
 
-@include 'http_api_options_client.mdx'
+The flags below should only be set if _no arguments_ are given. If no
+arguments are given, the flags below can be used to deregister a single
+service.
 
-#### Enterprise Options
+- `-id` - The ID of the service.
```

Comment:
```suggestion
- `-id` - Specifies the ID of a single service to deregister. Do not use this flag when providing a file containing multiple services that you want to deregister. 
```
The docs for this command are confusing, but I think this is what we mean. It was a lot of words/space for a single flag with one usage caveat.